src/Core/Framework/Update/Subscriber/UpdateSubscriber.php

```diff
@@ -51,18 +51,21 @@ public function updateFinishedDone(UpdatePostFinishEvent $event): void
             $integrationId = $source->getIntegrationId();
             $createdByUserId = $source->getUserId();
         }
+        $wrappedMessage = \wordwrap($event->getPostUpdateMessage(), 255, "\v", true);
```

Comment:
why not fix it on displaying side?

Reply:
Apparently I should've pushed this back in 2023, when I made this commit :D https://github.com/shopware/shopware/issues/4706 seems to have solved already. nvm

Reply:
:sweat_smile: so we can close this?

Reply:
you are a year to late :grin: https://github.com/shopware/shopware/commit/8e1c8b76dccc6e3f30a192e090fb4e17f945dd59